ticket_facts.view.lkml

```diff
@@ -0,0 +1,277 @@
+view: ticket_facts {
+  derived_table: {
+    sql: SELECT
+          audits.ticket_id,
+          tickets.status,
+          tickets.via__channel,
+          tickets.via__source__rel,
+          tickets.via__source__to__name,
+          groups.name,
+          metrics.group_stations AS number_of_groups_involved,
+          COUNT(DISTINCT audits.id) FILTER(WHERE audits.via__channel = 'voice' AND audits.via__source__from__name = 'Getaround'
+                                  AND audits.via__source__rel = 'outbound')  AS number_outbound_calls,
+          COUNT(DISTINCT audits.id) FILTER(WHERE (audits.via__channel = 'voice' AND audits.via__source__to__name = 'Getaround'
+                                  AND audits.via__source__rel = 'inbound')
+                                  OR (audits.via__channel = 'voice' AND audits.via__source__rel = 'voicemail'))  AS number_inbound_calls,
+          COUNT(DISTINCT audits.id) FILTER(WHERE (events.type = 'Comment' AND events.public = true AND audits.via__source__to__name = 'Getaround')
```

Comment:
What is `via__source__to_name = 'Getaround'` mean in the business context

Reply:
that the ticket is an inbound ticket to Getaround.  `Via` is an object telling us how the ticket was created, `source` is another object defining the source of the audit and `to_name` just explains where the event is going to.  
More documentation here: https://developer.zendesk.com/rest_api/docs/core/ticket_audits#the-via-object

This counts majority of our inbound emails.  I need to add two OR conditions for inbound email that come through the App and for tickets created by following up to a previously solved ticket.